README.md

```diff
@@ -66,8 +63,8 @@ After cloning this repository, build with following command.
 mvn package 
 ```
 
-Running Mini Instance
----------------------
+Running via Maven
```

Comment:
In c70a12d I commented out setting fluo do debug in log4j.props.  So these messages are no longer printed by default.